### PR TITLE
add: spread operator

### PR DESCRIPTION
**Pull request by infinityover**

### CODE REVIEW

File: src/markdown/spread-operator.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: "Spread Operator"
+subtitle: Operador de expansão dos itens de uma array
+infos: Mais conteúdo
+explanation: "Faz com que os elementos da array sejam expandidos para lugares onde são esperados 0 ou mais parâmetros"
```
Comment (LeonardoCesca) [inline src/markdown/spread-operator.md]: @infinityover 

Please, explain better `explanation`

File: src/components/layout/layout.js [hunk 2]
```diff
@@ -73,6 +73,7 @@ const Layout = ({ children }) => {
                 </SidebarItems>
                 <SidebarItems to="/if-else/">If Else</SidebarItems>
                 <SidebarItems to="/for/">Loop For</SidebarItems>
+                <SidebarItems to="/spread-operator/">Spread Operator</SidebarItems>
                 <CloseButton
                   onClick={() => setIsVisible(!isVisible)}
                 ></CloseButton>
```
Comment (LeonardoCesca) [inline src/components/layout/layout.js]: @infinityover ,

Please merged with `master` branch

Thank you!

Comment (infinityover) [reply src/components/layout/layout.js]: @LeonardoCesca, sorry i didn't understood.
This line have a problem, or you will merge it into the master branch of the project?

Comment (LeonardoCesca) [reply src/components/layout/layout.js]: @infinityover,

please, merge the master of project into your branch and push a new commit

Comment (infinityover) [reply src/components/layout/layout.js]: @LeonardoCesca Done.
Thanks for the support.

**Pull request merged**